test/test_models.py

```diff
@@ -189,18 +190,18 @@ def compute_mean_std(tensor):
 
             output = map_nested_tensor_object(out, tensor_map_fn=compact)
             prec = 0.01
-            strip_suffix = f"_{dev}"
             try:
                 # We first try to assert the entire output if possible. This is not
                 # only the best way to assert results but also handles the cases
                 # where we need to create a new expected result.
-                self.assertExpected(output, prec=prec, strip_suffix=strip_suffix)
+                self.assertExpected(output, name, prec=prec)
+                raise AssertionError
```

Comment:
@NicolasHug  I think this was left here by accident. This will cause the tests to be considered partially validated and marked as skipped.